src/Portal.js

```diff
@@ -26,9 +29,21 @@ class Portal extends React.Component {
     ])
   };
 
+  componentWillMount(){
+    if (useCreatePortal){
```

Comment:
this also needs a canUseDOM guard here, for serverside rendering. dom-helpers exports that constant for eas of use

Reply:
done